pages/1.12/storage/persistent-volume/index.md

```diff
@@ -451,7 +451,7 @@ The following example will create a pod with 2 containers and one shared persist
 
 ## Inspect and delete suspended stateful tasks
 
-To destroy and clean up persistent volumes and free the reserved resources associated with a task, perform 2 steps:
+Perform the following steps to free the reserved resources associated with a task, destroy and clean up persistent volumes. 
```

Comment:
The right order is, destroy and clean up persistent volumes first, and then free the reserved resources associated with a task. So I think the original content is correct in this regard.